test/Libraries/RevitIntegrationTests/SampleTests.cs

```diff
@@ -952,8 +952,8 @@ public void Revit_Floors_and_Framing()
             AssertNoDummyNodes();
 
             // check all the nodes and connectors are loaded
-            Assert.AreEqual(30, model.CurrentWorkspace.Nodes.Count);
-            Assert.AreEqual(34, model.CurrentWorkspace.Connectors.Count());
+            Assert.AreEqual(28, model.CurrentWorkspace.Nodes.Count);
```

Comment:
Yes, because we have hanged node which Structural Framing Node which earlier was taking 4 inputs and now it has 2 inputs only.